pcdsdevices/mirror.py

```diff
@@ -1263,29 +1308,6 @@ class FFMirrorZ(FFMirror):
     cool_press = Cpt(EpicsSignalRO, ':PRSM:1_RBV', kind='normal', doc="Axilon Panel Pressure Meter")
 
 
-class TwinCATMirrorStripe(TwinCATStatePMPS):
-    """
-    Subclass of TwinCATStatePMPS for the mirror coatings.
-
-    Unless most TwinCATStatePMPS, we have:
-    - Only in_states
-    - No in_states block the beam
-
-    We also clear the states_list and set _in_if_not_out to True
```

Comment:
I guess this isn't required anymore. @ZLLentz I didn't do this for the grating and the states were picked up with a problem.

Reply:
I think you're right- we can look closely later, but it isn't hurting anything so I'm not super eager to strip it all out